rust/arrow/src/array/builder.rs

```diff
@@ -1872,294 +1871,6 @@ impl FixedSizeBinaryBuilder {
     }
 }
 
```

Comment:
Wouldn't it be safer to deprecate the code (not sure what That's deprecation annotation is), then remove it 2 releases from now?

We don't yet have a Rust-specific deprecation policy, so I'm still fine if other opinions are to remove the builder immediately.

Reply:
I did not want to add new code without removing other code, but I am fine either way.